digits/templates/datasets/images/classification/show.html

```diff
@@ -89,7 +89,7 @@
         {% endautoescape %}
         );
     </script>
-    {% if task.mean_file %}
+    {% if (not job.status.is_running()) and task.mean_file %}
```

Comment:
I think you want something more like `task.status == 'D'` rather than `job.status.is_running()`, right? We can display the mean before the job is done as long as the task is done.